ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/recovery/RecoveryManager.java

```diff
@@ -223,6 +226,10 @@ public RecoveryManagerStatus trySuspend(boolean async) {
         }
     }
 
+    public RecoveryManagerStatus trySuspend(boolean async) {
```

Comment:
Please can I ask for clarification as to the reason this needs to be added to the public API for this class?

Reply:
Sorry, I see the reason now!